e2e/cypress/integration/account_settings/display/theme/custom_theme_sidebar_styles_spec.js

```diff
@@ -93,24 +93,24 @@ describe('Custom Theme - Sidebar Styles', () => {
         cy.get('#unreadIndicatorBottom').should('have.css', 'color', rgbArrayToString(themeRgbColor.mentionColor));
 
         // # Set user status to online
-        cy.userStatus(0);
+        cy.userStatus(1);
```

Comment:
While here, I'd like to take the opportunity to improve `cy.userStatus` command by deprecating it totally and replacing with the new command as `cy.uiOpenSetStatusMenu`:
```diff
--- a/e2e/cypress/support/ui/menu.js
+++ b/e2e/cypress/support/ui/menu.js
@@ -3,6 +3,7 @@
 
 const MAIN_MENU = 'main menu';
 const CHANNEL_MENU = 'channel menu';
+const SET_STATUS_MENU = 'set status';
 
 function openMenu(name, item) {
     const menu = () => cy.findByRole('button', {name}).should('be.visible');
@@ -45,3 +46,7 @@ Cypress.Commands.add('uiCloseChannelMenu', () => {
 Cypress.Commands.add('uiGetChannelMenu', () => {
     return getMenu(CHANNEL_MENU);
 });
+
+Cypress.Commands.add('uiOpenSetStatusMenu', (item = '') => {
+    return openMenu(SET_STATUS_MENU, item);
+});
```
Please update `menu.d.ts` as well.

Then this can be:
```suggestion
        cy.uiOpenSetStatusMenu('Online');
```

Other options could be: "Set a Custom Status", "Away", "Do Not Disturb" and "Offline"